fiftyone/core/collections.py

```diff
@@ -5540,6 +5540,77 @@ def export(
         if archive_path is not None:
             etau.make_archive(export_dir, archive_path, cleanup=True)
 
+    def annotate(
+        self,
+        backend="cvat",
+        label_schema=None,
+        label_field=None,
+        label_type=None,
+        classes=None,
+        attributes=True,
+        media_field="filepath",
+        launch_editor=False,
+        **kwargs,
+    ):
+        """Exports the samples and a label field to the given annotation
+        backend.
+
+        Args:
+            backend ("cvat"): the name of the annotation backend to which to
+                export the samples. Options are ("cvat")
+            label_schema (None): a dictionary indicating the type, class options, and
+                attributes for each label field. This is required for new label fields
+                if `classes` is not provided.
+                For existing label fields, provided classes and attributes will be used
+                instead of parsing existing classes or attributes
+            label_field (None): a string indicating either an existing label field to upload,
+                or the name of a new label field to create. Required if `label_schema` is not provided.
+            label_type (None): a string indicating the type of labels to expect 
+                when creating a new `label_field`. 
+                Options: ("detections", "classifications", "polylines", "keypoints", "scalar")
+            classes (None): a list of strings indicating the class options. These
+                classes will be used as the default for all fields without classes
+                specified in the `label_schema`. This is required for a new
+                `label_field` if `label_schema` is not provided. For existing label fields, if
+                neither `classes` nor `label_schema` is given, default classes are used
+                if available, otherwise classes are parsed from existing labels in the
+                label field
+            attributes (True): a list of string attributes or dictionary of attribute
+                name to type, values, and default values that will be the default for
+                every label field without attributes specified through the
+                `label_schema`. `True` indicates loading all values for existing
+                label fields. `False` indicates loading no attributes
```

Comment:
I'm a little confused here. What does "loading" mean? Attributes will be sent to the labeling backend? What does the `dict` options look like?

```
{ "attr_name": {"type": ..., values: [...], "default": ...}}
```

Reply:
FYI I updated the integration docs page. Hopefully this is explained better now